src/pmdas/amdgpu/GNUmakefile

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2014-2015,2020 Red Hat.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 2 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# for more details.
+#
+
+TOPDIR = ../../..
+include $(TOPDIR)/src/include/builddefs
+
+IAM	= amdgpu
+DOMAIN	= AMDGPU
+
+CMDTARGET = pmdaamd$(EXECSUFFIX)
+LIBTARGET = pmda_amd.$(DSOSUFFIX)
+CFILES	= localdrm.c amdgpu.c
+HFILES	= localdrm.h
+DFILES	= README
+LLDLIBS	= $(PCP_PMDALIB) $(LIB_FOR_DLOPEN) -ldrm -ldrm_amdgpu
```

Comment:
Depending on how the configure.ac machinery ends up (discussed elsewhere), the libraries listed here will likely end up accessed as makefile macros.